config/templates/stack-template.json

```diff
@@ -1116,5 +1116,25 @@
     {{end}}
     {{end}}
 
+  },
+
+  "Outputs": {
+    "WorkerSecurityGroup" : {
+      "Description" : "The security group assigned to worker nodes",
+      "Value" :  { "Ref" : "SecurityGroupWorker" },
+      "Export" : { "Name" : {"Fn::Sub": "${AWS::StackName}-WorkerSecurityGroup" }}
+    },
```

Comment:
Using existing `vpcId` will cause an error on `kube-aws up`: 

    Error: Failed to render stack template: invalid character '}' looking for beginning of object key string:
    json syntax error (offset=72048), in this region:
    -------
    }
        },
    
      }
    }

Quickly fixed by moving the comma from L:1126 to L1128

    "Export" : { "Name" : {"Fn::Sub": "${AWS::StackName}-WorkerSecurityGroup" }}
    }
    {{if not .VPCID}}
    ,
    "RouteTable" : {